meinberlin/apps/plans/assets/plans_map.jsx

```diff
@@ -439,7 +443,7 @@ class PlansMap extends React.Component {
                 <li>
                   <button
                     type="button"
-                    className="dropdown-item select-item"
+                    className="dropdown-item"
```

Comment:
note to myself: i should check the classes in the form widgets, category filters and this jsx filter